### ISSUE TITLE
leanpkg requires unique .lean file in package root

### ISSUE THREAD

**Issue by Kha** [leanprover/lean4#420]
People (including me) often create temporary .lean files in a package, which breaks leanpkg:
```
$ leanpkg build
...
share/lean/lean.mk:9: *** no unique .lean file found in current directory, please specify PKG. Stop.
```
We should automatically determine the root .lean file if there is a unique file that (case-insensitively) matches the package name, like we already do to [determine local imports](https://github.com/leanprover/lean4/blob/ccb873cea2b6b7d615f1e81dcd26d3b26d21ed48/src/Leanpkg.lean#L90-L91). If there is no such unique file, we should require a `root =` configuration variable or something. We can then pass the root file to `leanmake` via `PKG=`, or wait for #419 and do it there instead.

**Comment by leodemoura** [comment]
I hit this problem a few times in the part.

**Comment by gebner** [comment]
This should be resolved with #913, right?

**Comment by leodemoura** [comment]
@Kha Should we close this issue?
